src/App.js

```diff
@@ -1,23 +1,91 @@
-import logo from './logo.svg';
-import './App.css';
+import "./App.css";
+import React, { useState, useEffect } from "react";
+import Icon from "./components/Icon";
+import Input from "./components/Input";
+
+const getQueryUrl = (city) =>
+  `${process.env.REACT_APP_API_URL}?q=${city}&units=metric&appid=${process.env.REACT_APP_API_KEY}`;
+
+const getQueryUrlByCurrentLocation = (currentLocation) => {
```

Comment:
@pasternakewa use destructuring and put values from object directly to the string 
({latitude, longtitude})